lib/modules/apostrophe-doc-type-manager/index.js

```diff
@@ -80,15 +80,15 @@ module.exports = {
         withType: 'apostrophe-user',
         label: 'These Users can Edit',
         idsField: 'editUsersIds',
-        permission: 'admin'
```

Comment:
Give this a second look: does it do the right thing if doc type manager is the parent class of a piece rather than a page and the piece type happens to have edit permissions enabled?